aiobungie/crates/character.py

```diff
@@ -211,7 +211,7 @@ class Character:
     race: enums.Race
     """Character's race"""
 
-    emblem: assets.Image
+    emblem: assets.Image | None
```

Comment:
@spacez320 There's no need to make the `Image` on the character object optional ***for now***,

`Image` itself takes optional `str | None` and return the default URL if it was `None`, So `Image(None)` will return the default Bungie icon URL.

Just revert this. We won't touch `Character` for now.
```suggestion
    emblem: assets.Image
```